### PR TITLE
DB seeding and tests

### PR DESCRIPTION
**Pull request by jemherre**
I am bad at creating variable names, so if something can be rewritten to be worded better would be amazing.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const knex = require('../knex/knex.js');
```
Comment (clifton311) [inline database/index.js]: I am not too familiar with knex, but it looks good from what i can tell. I think you should elaborate more on the error on the catch

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  var id = Number(req.params.id);
+  console.log('reqP: ', req.params, 'id: ', id);
+  db.getImg(id, (data)=>{
+    console.log('D: ', data);
+    res.send(data);
+  });
+});
+
+app.post('/gallery/update/:order',(req, res) => {
+  console.log('reqP: ', req.params, 'id: ', order);
```
Comment (clifton311) [inline server/index.js]: maybe get rid of the console.logs

Comment (jemherre) [reply server/index.js]: yes, definitely will clean out my code and get rid of the console.logs . Thank you!

File: database/index.js (hunk 1 above)
Comment (jemherre) [reply database/index.js]: I added a more specific message for each function.

**Pull request merged**